benchmarks/linear-elastic-plate-with-hole/workflow_creation_guide.md

```diff
@@ -0,0 +1,930 @@
+- Multiple parameter configurations per benchmark
+- Automated mesh generation
+- Standardized output formats
+- Comprehensive provenance tracking and metadata collection
+- Automated results summarization and visualization
+
+## Benchmark Structure Requirements
+
+Create a dedicated directory for your benchmark with the following structure:
+
+```
+your-benchmark-name/
+├── README.md                        # Documentation
+├── generate_config.py               # Configuration generator
+├── workflow_config.json             # Generated workflow configuration
+├── create_mesh.py                  # Mesh generation script
+├── summarise_results.py            # Results aggregation script
+├── parameter_extractor.py          # Provenance parameter extraction
+├── plot_provenance.py              # Provenance visualization
+├── plateWithHoleSolution.py        # Analytical solution (if applicable)
+├── parameters_*.json               # Parameter files for each configuration
+├── environment_mesh.yml            # Conda environment for meshing
+├── environment_postprocessing.yml  # Conda environment for post-processing
+│
+├── # Snakemake Implementation
+├── Snakefile                       # Main Snakemake workflow
+│
+├── # Nextflow Implementation  
+├── main.nf                         # Main Nextflow workflow
+├── nextflow.config                 # Nextflow configuration
+│
+└── tool1/                          # Directory for simulation tool 1
+    ├── Snakefile                   # Tool-specific Snakemake workflow
+    ├── tool1.nf                    # Tool-specific Nextflow workflow
+    ├── environment_simulation.yml  # Tool-specific conda environment
+    ├── run_tool1_simulation.py     # Main simulation script
+    ├── create_tool1_input.py       # Input file preparation (if needed)
+    ├── postprocess_results.py      # Post-processing script (if needed)
+    └── template_files/             # Template configuration files
+```
+
+## Algorithm for Creating Dual-System Workflows
+
+### Step 1: Define Parameter Files and Configuration
+
+Create parameter files following the standardized JSON format:
+
+```json
+{
+    "configuration": "unique_config_name",
+    "geometry_parameters": {
+        "radius": {"value": 0.33, "unit": "m"},
+        "length": {"value": 1.0, "unit": "m"}
+    },
+    "material_properties": {
+        "young-modulus": {"value": 210e9, "unit": "Pa"},
+        "poisson-ratio": {"value": 0.3, "unit": ""}
+    },
+    "mesh_parameters": {
+        "element-size": {"value": 0.1, "unit": "m"},
+        "element-order": 1,
+        "element-degree": 1
+    },
+    "solver_settings": {
+        "quadrature-rule": "gauss",
+        "quadrature-degree": 1
+    },
+    "loading_conditions": {
+        "load": {"value": 100.0, "unit": "MPa"}
+    }
+}
+```
+
+### Step 2: Implement Configuration Generator
+
+Create `generate_config.py` to automatically discover and validate configurations:
+
+```python
```

Comment:
don't copy the files, rather make a link and document important parts, maybe even do the documentation directly in the linked file, than it will automatically be updated when someone is changing something.